### DISCUSSION TITLE
Dreame D10s Plus Avoiding Area Rug

### DISCUSSION THREAD

**Discussion by stgdcg** [Hypfer/Valetudo#2119] [Q&A / Support] [answered]
### Are you already using Valetudo?

Yes, just started

### Your questions

I've recently installed Valetudo on my Dreame D10s Plus & it isn't cleaning my area rug. It cleans it when I manually start it using the button on the robot but it won't when I start it from Valetudo. I've tried disabling carpet mode & obstacle avoidance but that doesn't seem to affect it. I've also checked on swagger & the mop is correctly reading as detached. Any ideas on what's going wrong?

**Comment by stgdcg** [comment, marked as answer]
This was fixed by taping some paper over the cliff sensors. The rug was black & that was confusing the sensors.

Is there a way to disable the cliff sensors either entirely or for a specific area?

**Reply by Hypfer** [reply]
Unfortunately no. Probably because the vendors don't want people to RMA robots that presentably fell down the stairs.
I did try playing around with patching out the checks in the software but then hit what I assume was a failsafe in the MCU that too brought the robot to a stop when the cliff sensors triggered; just more violently.
